packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -1146,14 +1146,30 @@ export default class BackupNg {
         $defer.call(xapi, 'deleteVm', snapshot)
       }
 
+      const compress = getJobCompression(job)
+      const { $pool: pool } = snapshot
+      if (
+        compress === 'zstd' &&
+        pool.restrictions.restrict_zstd_export !== 'false'
+      ) {
+        logger.warning(
+          `Zstd is not supported on the pool ${
+            pool.name_label !== '' ? pool.name_label : pool.uuid
```

Comment:
Why?

Reply:
The pool can not have the label and we should know which pool not support the zstd compression. I can only show the pool uuid.

Reply:
If the user chose to name their pool `''`, it's their choice. Why showing something different than the name the user chose?

Reply:
If the user not set the pool name its value will be `' '`, the warning should identify the concerned pool.

Reply:
What do you mean? The user chooses the name of the pool (of the host thus the pool) when he installs it.